spec/controllers/idv/review_controller_spec.rb

```diff
@@ -0,0 +1,148 @@
+require 'rails_helper'
+
+describe Idv::ReviewController do
+  render_views
+
+  let(:user) { create(:user, :signed_up, email: 'old_email@example.com') }
```

Comment:
Yes, `build_stubbed`, but that uses the `factory_girl` gem. Why invoke more code than necessary? The User does not need to have any specific attributes for this test.